### PR TITLE
feat: add feedback to smart switch group interactions

### PR DESCRIPTION
**Pull request by psykzz**
Adds a response to the custom commands for smart switch groups
Additionally adds a status to the base command.

### CODE REVIEW

**Comment by alexemanuelol** [comment]
Nice one! I like it!
Please change on/off to ON/OFF, and use /**/ comments.

File: src/handlers/inGameCommandHandler.js [hunk 1]
```diff
@@ -168,13 +168,15 @@ module.exports = {
                         rustplus.printCommandOutput(`Turning ${groupName} off.`);
```
Comment (alexemanuelol) [inline src/handlers/inGameCommandHandler.js:168]: These off/on to OFF/ON as well, squash the commits into one and it should be good to go.

Comment (psykzz) [reply src/handlers/inGameCommandHandler.js:168]: You can squash when you merge, just select "[Squash & Merge](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/merging-a-pull-request#merging-a-pull-request)".

File: src/handlers/inGameCommandHandler.js [hunk 2]
```diff
@@ -87,15 +87,15 @@ module.exports = {
                     if (command === cmd) {
                         active = !content.active;
                     }
-                    else if (command === `${cmd} on`) {
+                    else if (command === `${cmd} ON`) {
```
[the remark this answers is not in the document]
Comment (psykzz) [reply src/handlers/inGameCommandHandler.js:90]: Oh crap my mistake - fixing now.

**Pull request merged**